toolium/selenoid.py

```diff
@@ -66,6 +67,11 @@ def __init__(self, driver_wrapper, **kwargs):
         self.output_directory = kwargs.get('output_dir', DriverWrappersPool.output_directory)
         self.browser_remote = driver_wrapper.config.getboolean_optional('Server', 'enabled', False)
         self.enabled_logs = driver_wrapper.config.getboolean_optional('Server', 'logs_enabled', False)
+        self.browser = driver_wrapper.config.get('Driver', 'type')
```

Comment:
use driverwrapper.driver.desired_capabilities['browserName'] instead of driver/type, it contains final browser name, like 'internet explorer'

Reply:
Fixed in a07aac3